src/librustc_mir/shim.rs

```diff
@@ -198,20 +198,13 @@ fn build_drop_shim<'tcx>(tcx: TyCtxt<'tcx>, def_id: DefId, ty: Option<Ty<'tcx>>)
     block(&mut blocks, TerminatorKind::Goto { target: return_block });
     block(&mut blocks, TerminatorKind::Return);
 
-    let mut body = Body::new(
-        blocks,
-        IndexVec::from_elem_n(
-            SourceScopeData { span: span, parent_scope: None }, 1
-        ),
-        ClearCrossCrate::Clear,
-        None,
-        local_decls_for_sig(&sig, span),
-        IndexVec::new(),
-        sig.inputs().len(),
-        vec![],
-        span,
-        vec![],
-    );
+    let mut body = new_body(blocks,
```

Comment:
I like the previous formatting (block indent) more because it causes less churn in later changes